frontend/src/constants/schema.tsx

```
@@ -141,46 +141,78 @@ export const SIGNUP_FORM_SCHEMA = yup.object({
       /[@$!%*?&#]/,
       'Password must contain at least one special character (@, $, !, %, *, ?, &, or #).',
     ),
-  confirmPassword: yup
-    .string()
-    .when('newPassword', {
-      is: (newPassword: string) => newPassword && newPassword.length > 0,
-      then: (schema) => schema
+  confirmPassword: yup.string().when('newPassword', {
+    is: (newPassword: string) => newPassword && newPassword.length > 0,
+    then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)